docs/querying/sql-functions.md

```diff
@@ -1282,22 +1282,75 @@ Returns the following:
 
 ## BLOOM_FILTER
 
-Computes a Bloom filter from values produced by the specified expression.
+Computes a [Bloom filter](../development/extensions-core/bloom-filter.md) from values provided in an expression.
 
-* **Syntax**: `BLOOM_FILTER(expr, <NUMERIC>)`
+
+* **Syntax:** `BLOOM_FILTER(expr, numEntries)`  
+  `numEntries` specifies the maximum number of distinct values before the false positive rate increases.
 * **Function type:** Aggregation
 
+<details><summary>Example</summary>
+
+The following example returns a Base64-encoded Bloom filter string for entries in `agent_category`:
+
+```sql
+SELECT
+  agent_category,
+  BLOOM_FILTER(agent_category, 10) as bloom
+FROM "kttm"
+  GROUP BY agent_category
+```
+
+Returns the following:
+
+| `agent_keys` | `bloom` |
```

Comment:
Suggest changing the example as suggested in line 1294, but note that the SELECT is on `agent_category` and the results column is `agent_keys`.